components/user_profile/index.ts

```diff
@@ -5,14 +5,16 @@ import {connect} from 'react-redux';
 
 import {getUser, makeGetDisplayName} from 'mattermost-redux/selectors/entities/users';
 
-import UserProfile from './user_profile';
+import {GlobalState} from 'mattermost-redux/types/store';
+
+import UserProfile, {UserProfileProps} from './user_profile';
 
 function makeMapStateToProps() {
     const getDisplayName = makeGetDisplayName();
 
-    return (state, ownProps) => {
+    return (state: GlobalState, ownProps: UserProfileProps) => {
         return {
-            displayName: getDisplayName(state, ownProps.userId),
+            displayName: getDisplayName(state, ownProps.userId, true),
```

Comment:
If this value wasn't provided, a `boolean` here would default to `false` I believe, unless this is some special case.

Reply:
Default values strike again!

https://github.com/mattermost/mattermost-redux/blob/master/src/selectors/entities/users.ts#L618

Reply:
I see.

Reply:
Glad to see that my intuition was eventually correct. I did a bit of digging and ended up exactly on that line mentioned by @hmhealey and I assumed the default was `true`. Great to have confirmation. Thanks